### ISSUE TITLE
Abandoned?

### ISSUE THREAD

**Issue by woodzza** [kitolog/cordova-plugin-screen-locker#8]
Has this project been abandoned? .. I need to be able to initiate the devices default lock from without my Cordova based app.

**Comment by kitolog** [comment]
Yes, I'm not worked with cordova for 2 years. If you want to make updates and pull requests, you are welcome

**Comment by woodzza** [comment]
thanks for the response .. does the plugin work at all?... I am looking for a plugin that just LOCKS the phone using the native phone lock (android only)
